### PR TITLE
Add /content/fr/docs/reference/_index.md

### PR DESCRIPTION
**Pull request by daverick**
Add `/content/fr/docs/reference/_index.md`

Without this file, the automatic navigation menus were broken.
Simple Translation of the `/content/en/docs/reference/_index.md`

### CODE REVIEW

File: content/fr/docs/reference/_index.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+title: Reference
+approvers:
+- chenopis
```
Comment (rekcah78) [inline content/fr/docs/reference/_index.md]: il faut supprimer le bloc "approvers"

File: content/fr/docs/reference/_index.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+* [kubelet](/docs/admin/kubelet/) - Le principal *agent* qui s'exécute sur chaque noeud. Kubelet prends un ensemble de PodSpecs et s'assure que les conteneurs qui y sont décrit s'exécutent correctement.
+* [kube-apiserver](/docs/admin/kube-apiserver/) - L'API REST qui valide et configure les données des objects de l'API tels que les Pods, Services, Deployments ...
+* [kube-controller-manager](/docs/admin/kube-controller-manager/) - Démon en charge de la principale boucle de régulation (core control loop) de Kubernetes.
+* [kube-proxy](/docs/admin/kube-proxy/) - S'occupe de reacheminer (forwarding) simplement les flux TCP/UDP ou bien en utilisant un Round-Robin sur un ensemble de back-ends.
+* [kube-scheduler](/docs/admin/kube-scheduler/) - Ordonnanceur (scheduler) qui gère la disponibilité, la performance et la capacité.
+* [federation-apiserver](/docs/admin/federation-apiserver/) - Serveur API pour les clusters fédérés.
+* [federation-controller-manager](/docs/admin/federation-controller-manager/) - Démon en charge de la boucle de régulation (core control loop) d'une fédération de clusters Kubernetes.
+
+## Documents de design
```
Comment (rekcah78) [inline content/fr/docs/reference/_index.md]: Documents de conception

File: content/fr/docs/reference/_index.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+* [Architecture de Kubernetes](https://git.k8s.io/community/contributors/design-proposals/architecture/architecture.md)
+* [Vue d'ensemble des documents de Design de Kubernetes](https://git.k8s.io/community/contributors/design-proposals).
```
Comment (rekcah78) [inline content/fr/docs/reference/_index.md]: des documents de conception de Kubernetes

File: content/fr/docs/reference/_index.md (hunk 1 above)
Comment (daverick) [reply content/fr/docs/reference/_index.md]: ok

File: content/fr/docs/reference/_index.md (hunk 3 above)
Comment (daverick) [reply content/fr/docs/reference/_index.md]: merci

File: content/fr/docs/reference/_index.md (hunk 2 above)
Comment (daverick) [reply content/fr/docs/reference/_index.md]: ok

File: content/fr/docs/reference/_index.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+* [Kubernetes Go client library](https://github.com/kubernetes/client-go/)
+* [Kubernetes Python client library](https://github.com/kubernetes-client/python)
+* [Kubernetes Java client library](https://github.com/kubernetes-client/java)
+* [Kubernetes JavaScript client library](https://github.com/kubernetes-client/javascript)
+
+## Documents de Référence des outils en ligne de commande (CLI)
+
+* [kubectl](docs/reference/command-line-tools-reference/kubelet/) - Principal outil en ligne de commande (CLI) pour exécuter et gérer un cluster Kubernetes.
+    * [JSONPath](/docs/user-guide/jsonpath/) - Guide de la syntaxe des [ expressions JSONPath](http://goessner.net/articles/JsonPath/) avec kubectl.
```
Comment (sftim) [inline content/fr/docs/reference/_index.md]:
```suggestion
    * [JSONPath](/docs/user-guide/jsonpath/) - Guide de la syntaxe des [expressions JSONPath](http://goessner.net/articles/JsonPath/) avec kubectl.
```
(whitespace in Markdown)

Comment (sftim) [inline content/fr/docs/reference/_index.md]:
```suggestion
    * [JSONPath](/docs/user-guide/jsonpath/) - Guide de la syntaxe des [expressions JSONPath](http://goessner.net/articles/JsonPath/) avec kubectl.
```
(whitespace in Markdown)

**Pull request merged**